lib/SILOptimizer/Mandatory/TFDeabstraction.cpp

```diff
@@ -2350,34 +2346,29 @@ void TFDeabstraction::evaluateAttributesAndDoPacking(
           if (unpackTensorAggregates(context, origInst->getLoc(), B, element,
```

Comment:
(github won't let me comment on L2342 so I'm commenting on the closest possible line)

Could you also delete the error when `!element`? I don't remember what causes that situation and I don't remember if we have any test cases exercising it. (Maybe arrays that are constructed using global vars as elements?) It seems like it should be fine to fall back to eager mode in that case too.

Reply:
Done!